### ISSUE TITLE
Arbitrary alphabet conversion?

### ISSUE THREAD

**Issue by Altreus** [thundergnat/Base-Any#1]
This would be cool:

    123456.to-base-alphabet('0123456789abcdefghjkmnpqrsuwxz')

This would simply take the number of characters as the base, but override the default set of digits for that base.

That way I can selectively remove certain symbols from the default set (and alter the default set for >10).

My use-case is to render a matrix of integers on the terminal, and so I want to be able to choose recognisable characters for each "tile".

**Comment by Altreus** [comment]
I made a start but I don't undersand the Real number conversion :D works for ints though

https://github.com/Altreus/Base-Any/tree/alphabet-conversion

**Comment by Altreus** [comment]
Updated to accept either an array or a string for the alphabet :)

**Comment by thundergnat** [comment]
Hi Altreus, your suggestion is a good one, and one I had planned to implement eventually, but hadn't had the motivation to go ahead and do it. I have pushed a new version with custom digit set loading. I took a different approach than yours, rather than adding multi subs that accept a custom digit set, I added the capability to just swap out the default digit set and re-use all the original (with some minor tweaks) routines. Some routines are too fiddly (imaginary base routines) to easily modify to work with possibly "out-of-order" digit glyphs, but they are of questionable value other than as a curiosity anyway. See if v0.1.0 solves your problem.

Cheers!
